src/analyserequation_p.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
What is the _p in the filename for?

Reply:
It stands for private. It's a convention that is used in some projects (e.g. Qt). I use that mechanism because some Analyser-related classes are `friend class`es of `Analyser` (and `Generator` in a specific case). I am not a fan of friend classes, but there is no way around it due to our use of the PIMPL idiom (unless we were to put all of our Analyser-related classes in one humongous `.h` and `.cpp` file, which wouldn't be neat).